pkg/dashboard/auth/iguazio/auth.go

```diff
@@ -59,8 +59,13 @@ func (a *Auth) Authenticate(request *http.Request) (auth.Session, error) {
 		"cookie":        cookie,
 	}
 
+	url := a.config.Iguazio.VerificationURL
+	if options.EnrichDataPlane && a.config.Iguazio.VerificationDataEnrichmentURL != "" {
```

Comment:
What if `options.EnrichDataPlane == true` and `a.config.Iguazio.VerificationDataEnrichmentURL == ""`?
Is this a valid configuration? Or should we explode in this case?